dev/gpctl/cmd/workspaces-last-heartbeat.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/gitpod-io/gitpod/ws-manager/api"
+)
+
+// workspacesLastHeartbeatCmd get workspace last heartbeat time
+var workspacesLastHeartbeatCmd = &cobra.Command{
+	Use:   "last-heartbeat <instanceID>",
+	Short: "get workspace last heartbeat time",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		conn, client, err := getWorkspacesClient(ctx)
+		if err != nil {
+			log.WithError(err).Fatal("cannot connect")
+		}
+		defer conn.Close()
+
+		instanceID := args[0]
```

Comment:
How about receiving get multiple instances IDs?

Reply:
That probably can be done as a separate PR. But good suggestion.

Reply:
I'm not going to create a follow-up PR to support multiple instance ids since our commands like `snapshot` `stop` `backup` all only works for one instance. 

We can do it like for now
```
cat backup-pods.txt | xargs -n1 -I{} gpctl workspaces backup {}
```